app/move/controllers/create/document.js

```diff
@@ -1,24 +1,46 @@
-const formidable = require('formidable')
-const { get } = require('lodash')
+const multer = require('multer')
+const { reject } = require('lodash')
 
-const documentService = require('../../../../common/services/document')
-const moveService = require('../../../../common/services/move')
 const CreateBaseController = require('./base')
-const { API, FEATURE_FLAGS } = require('../../../../config')
+const documentService = require('../../../../common/services/document')
+const { FILE_UPLOADS, FEATURE_FLAGS } = require('../../../../config')
+
+const upload = multer({
+  dest: FILE_UPLOADS.UPLOAD_DIR,
+  limits: {
+    fileSize: FILE_UPLOADS.MAX_FILE_SIZE,
+  },
+})
+
+// TODO:
+// - bug with simulteaneous xhr uploads
```

Comment:
? It's a bit cryptic / scary :-) Maybe add a reference to a Jira ticket?

Reply:
Good call. Am going to move to a ticket instead.

Reply:
[P4-976](https://dsdmoj.atlassian.net/browse/P4-976)